docs/create_your_first_nn.md

```diff
@@ -165,6 +165,123 @@ Total params: 266610
 ==============================================================================
 
 ```
-Great! We have now defined the structure of our neural network, specified the optimization algorithm that will be used during its training, and identified accuracy as the metric we will use to gauge its success.
 
-In the [following tutorial](training_a_model.md), you'll learn how to train this model on the [Fashion-MNIST dataset](https://github.com/zalandoresearch/fashion-mnist).
+## Training a model
+
+In the previous sections we have defined the structure of our neural network, specified the optimization algorithm that will be used during its training, and identified accuracy as the metric we will use to gauge its success.
+
+In this section you'll learn how to train this model on the [Fashion-MNIST dataset](https://github.com/zalandoresearch/fashion-mnist).
+
+Before you can use data, typically some preprocessing is required.
```

Comment:
Before you can use data, typically, some preprocessing is required.
In this case, it's minimal – all the images are already the same size and are grayscale.
With the built-in functionality, we can convert the [Fashion MNIST image archives](https://github.com/zalandoresearch/fashion-mnist#get-the-data) into a dataset object **used** for model training.